.travis.yml

```diff
@@ -1,18 +1,59 @@
 ---
+# Add additional stages in the order of execution here, and then under the job:include: key
+dist: "focal"  # Ubuntu 20.04
+stages:
+  - name: "lint"
+  - name: "build"
+  - name: "test"
+  - name: "integration"
+  # - name: "deploy"
+  #   if: "branch = master"
+
 language: "python"
-services:
-  - "docker"
 python:
   - 3.7
   - 3.8
 
-before_script:
-  - "pip install invoke toml poetry ansible pynetbox"
```

Comment:
Maybe I didn't follow far enough but why do we need ansible?

Reply:
pip install poetry?

Reply:
we are using ansible during the integration tests to populate the initial inventory in Netbox